README.md

```diff
@@ -45,55 +45,69 @@ The following are also relevant for consistency, but not critical:
 
 ## Backup
 
-To backup, prepare to receive the backup data in the terminal:
-
-```sh
-adb reverse tcp:12101 tcp:12101
-nc -l -s 127.0.0.1 -p 12101 > firefox-android-backup.tar.gz
-```
-
-Open the Firefox app on your phone, and visit `about:blank` (any webpage works).
-Copy the contents of [`snippets_for_firefox_debugging.js`](snippets_for_firefox_debugging.js).
-Open `about:debugging`, scroll down to "Main Process" and click on Inspect.
-Switch to the Console, paste the code and run it. Then type and run:
-
-```js
-fab_backup_create();
-```
-
-This copies [relevant files](#relevant-files) to `firefox-android-backup.tar.gz`
-without changing any data, except for one log file. To remove the log file, run:
-
-```js
-fab_cleanup();
-```
+Optionally before performing the backup clear the app cache of the Firefox app
+to reduce the size of the backup.
+
+1. To backup, prepare to receive the backup data in the terminal:
+    ```sh
+    adb reverse tcp:12101 tcp:12101
+    nc -l -s 127.0.0.1 -p 12101 > firefox-android-backup.tar.gz
+    ```
+
+    On MacOS you might have to use `nc -l 127.0.0.1 12101 > ...` ([issue #3](https://github.com/Rob--W/firefox-android-backup-restore/issues/3)).\
+    On Windows you can use [ncat](https://nmap.org/ncat/) with `ncat -l 127.0.0.1 -p 12101 > ...`.
```

Comment:
The idea to use `ncat` (and also to clear the cache before the backup) is from [this Reddit comment](https://www.reddit.com/r/firefox/comments/1ev2ao8/comment/ln7wf5h).